mail_sent/models.py

```diff
@@ -16,6 +16,15 @@ def _get_sent(self):
                          and self_sudo.author_id \
                          and self_sudo.partner_ids[0].id != self_sudo.author_id.id
 
+    @api.multi
+    def message_format(self):
+        message_values = super(MailMessage, self).message_format()
+        for message in message_values:
+            for item in self:
+                if message['id'] == item.id:
+                    message['sent'] = item.sent
+        return message_values
```

Comment:
Optimize this someway, e.g.

        message_index = {}
        for message in message_values:
            message_index[message['id']] = message
        for item in self:
            msg = message_index.get(item.id)
            if msg:
                    message['sent'] = item.sent